lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -434,7 +440,12 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            install_puppetlabs_release_repo(host)
+            if host[:type] == 'aio'
+              install_puppetlabs_release_repo(host,'pc1',opts)
+              opts[:version] = 'agent' unless opts[:version]
```

Comment:
it seems to me that there's an assumption that `opts[:version]` will be a version number. Can you point to a place where we're breaking that such that `agent` would be an appropriate value for this field?